packages/component-elife-ui/client/ui/molecules/BurgerMenu.js

```diff
@@ -5,10 +5,10 @@ import { Box, Flex } from '@rebass/grid'
 import { th } from '@pubsweet/ui-toolkit'
 import NavLink from '@elifesciences/component-elife-ui/client/ui/atoms/NavLink'
 import media from '@elifesciences/component-elife-ui/client/global/layout/media'
-import Icon from '../atoms/Icon'
-import ButtonAsIconWrapper from '../atoms/ButtonAsIconWrapper'
-import CrossIconButton from '../atoms/CrossIconButton'
-import ModalOverlay from '../molecules/ModalOverlay'
+import Icon from '@elifesciences/component-elife-ui/client/ui/atoms/Icon'
```

Comment:
now the #1781 was in, the ui folder has to be removed

Reply:
I've just merged develop branch will push an update soon